src/main.rs

```diff
@@ -658,27 +658,24 @@ fn get_setup(args: &[String]) -> Setup {
         }
     }
 
-    let cmd_args = &args[1..];
+    let args_len = args.len();
 
-    let cmd_args_len = cmd_args.len();
-
-    if cmd_args_len > 0 {
+    if args_len > 1 {
```

Comment:
`!args.is_empty()` would probably be a bit more idiomatic here.

Reply:
We don't want to know if `args.is_empty()`. arg will never be empty. it will always at least contain the path to the `librespot` binary. It will always have a length of at least 1. We are specifically asking it contains more than that, i.e. actual args passed to the binary.

I also lying. I have one more commit that logs an error when we run into non-Unicode env vars. Currently they're ignored silently.

Reply:
Ah of course.